fuel/converters/adult.py

```diff
@@ -0,0 +1,107 @@
+import os
+
+import h5py
+import numpy
+
+from fuel.converters.base import fill_hdf5_file
+
+
+def convert_adult(directory, output_directory,
+                  output_filename='adult.hdf5'):
+    """Convert the Adult dataset to HDF5.
+
+    Converts the Adult dataset to an HDF5 dataset compatible with
+    :class:`fuel.datasets.Adult`. The converted dataset is saved as
+    'adult.hdf5'.
+    This method assumes the existence of the file `adult.data` and
+    `adult.test`.
+
+    Parameters
+    ----------
+    directory : str
+        Directory in which input files reside.
+    output_directory : str
+        Directory in which to save the converted dataset.
+    output_filename : str, optional
+        Name of the saved dataset. Defaults to `None`, in which case a name
+        based on `dtype` will be used.
+
+    Returns
+    -------
+    output_paths : tuple of str
+        Single-element tuple containing the path to the converted dataset.
+    """
```

Comment:
PEP257 asks that you put a blank line right before the end of a multi-line docstring, like so:

``` python
"""My docstring

Parameters
----------
param : int
    Some description

"""
```

This should solve one of your two Scrutinizer errors.